docs/src/main/asciidoc/se/health.adoc

```diff
@@ -492,9 +492,9 @@ specification is also provided for the Kubernetes service and deployment.
 ----
 include::{sourcedir}/se/HealthSnippets.java[tag=snippet_8, indent=0]
 ----
-<1> The health service for the `liveness` probe is exposed at `/observe/health/live`.
+<1> The health service for the `liveness` probe is exposed at `/health/live`.
 <2> Using the built-in health checks for the `liveness` probe.
-<3> The health service for the `readiness` probe is exposed at `/observe/health/ready`.
+<3> The health service for the `readiness` probe is exposed at `/health/ready`.
```

Comment:
If this is SE with no config changes to alter the default endpoint paths, then the `/observe` prefix _should_ be present.

Reply:
See comment above. This text is annotating a code snippet that changes the default endpoint path to "/health". The bug is that the text annotations did not match the code snippet.

Reply:
Ugh. Sorry.